ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -567,12 +596,12 @@ def test_torch_moveaxis(
     keepdim=st.booleans(),
 )
 def test_torch_max(
-    dtype_input_axis,
-    as_variable,
-    num_positional_args,
-    native_array,
-    keepdim,
-    with_out,
+        dtype_input_axis,
```

Comment:
These indentations were not necessary
```suggestion
    dtype_input_axis,
```